### ISSUE TITLE
Error from server (AlreadyExists): project.project.openshift.io "test-project" already exists

### ISSUE THREAD

**Issue by vasukulkarni** [red-hat-storage/ocs-ci#345]
failed with -m tier1 run

```
        #log.debug(f"Command output: {r.stdout.decode()}")
        if r.stderr and not r.returncode:
            log.warning(f"Command warning:: {r.stderr.decode()}")
        if r.returncode:
            raise CommandFailed(
>               f"Error during execution of command: {cmd}."
                f"\nError is {r.stderr.decode()}"
            )
E           ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: ['oc', 'new-project', 'test-project'].
E           Error is Error from server (AlreadyExists): project.project.openshift.io "test-project" already exists
```

**Comment by vasukulkarni** [comment]
and the same test script before failed with

```
tests/manage/test_pvc_concurrent_deletion_creation.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <class 'tests.manage.test_pvc_concurrent_deletion_creation.TestMultiplePvcConcurrentDeletionCreation'>

    def teardown(self):
        """
        Delete PVCs
        Delete project
        """
        # Delete newly created PVCs
        command = (
            f'for i in `seq 1 {self.number_of_pvc}`;do oc delete pvc '
            f'{self.pvc_base_name_new}$i -n {TEST_PROJECT};done'
        )
        proc = run_async(command)
        assert proc, (
            f'Failed to execute command for deleting {self.number_of_pvc} PVCs'
        )
    
        # Verify command to delete PVCs
        ret, _, _ = proc.async_communicate()
>       assert not ret, 'Deletion of newly created PVCs failed'
E       AssertionError: Deletion of newly created PVCs failed
E       assert not 1
```

**Comment by jilju** [comment]
This will be fixed as an additional requirement in PR #316.
